assets/js/opensuse-theme.js

```diff
@@ -252,6 +253,19 @@ $(document).on("ready", function(){
   }
 
 });
+
+//change page direction when changing lang into/from ar
+$(window.lang).on("afterUpdate", function (event, currentLang, newLang, pack1, pack2) {
```

Comment:
why are you passing the arguments event, pack1 and pack2 ? I dont see where you are using them below